### DISCUSSION TITLE
[User Cases] Welcome to leave your company‘s info here

### DISCUSSION THREAD

**Discussion by luzhijing** [apache/doris#27683] [A - General / Q&A]
### 👏 Welcome! 

Apache Doris now has a user base of over 3w+ data engineers from more than 4000 enterprise users. This vibrant and diverse community comprises professionals from various sectors , including technology, finance, telecommunications, manufacturing, logistics and retail. 

The community deeply appreciates every user and developer who uses and cares about Doris. We are dedicated to listening to each one of you and to stick community innovation. 

To kickstart this topic, we encourage you, as a user or developer, to share your company information and real-world scenarios. This will enable us to collectively explore new and improved solutions, and address real-world challenges effectively. 

We will periodically sync information with the 🔗[official website](https://doris.apache.org/users) and the community eagerly awaits newcomers as always.

---
Here is the template:
Your company: Xiaomi
Website: [Xiaomi](https://www.mi.com/)
User case: Xiaomi Group adopted Apache Doris for the first time in 2019 due to the need for growing analysis business. After four years of development, Apache Doris has been widely used in dozens of businesses such as A/B tests, user behavior analysis, advertising BI, Xiaomi Youpin, and Xiaomi Auto, with dozens of clusters and hundreds of nodes. In the future, we will bring Apache Doris to more core business scenarios, and gradually upgrade to the newest version.

Logo: 
<img src="https://github.com/apache/doris/assets/82810928/89ef394f-57f0-41c9-afeb-2022f36c2daf" width="20%" height="auto">

**Comment by luzhijing** [comment]
Your company: NetEase Games
Website: [https://www.neteasegames.com/](https://www.neteasegames.com/)
User case: 

Logo: 
<img src="https://user-images.githubusercontent.com/24907215/174302370-e5af925b-9ca5-4550-be73-ebb1782db9cb.png" width="40%" height="auto">

**Comment by luzhijing** [comment]
Your company: 物易云通
Website: [https://www.wyyt.com/](https://www.wyyt.com/)
User case: 

Logo: 

<img src="https://user-images.githubusercontent.com/14903509/174304189-7e77ac5c-b7a4-4937-a81f-cf552f08c93d.png" width="40%" height="auto">

**Comment by luzhijing** [comment]
Your company: 同程数科
Website: [https://jr.ly.com/](https://jr.ly.com/)
User case: 

Logo: 
<img src="https://user-images.githubusercontent.com/52272263/174304314-1cf175b5-226c-491a-a164-5e40622e312f.png" width="50%" height="auto">

**Comment by luzhijing** [comment]
Your company: Footprint Analytics
Website: [https://www.footprint.network/](https://www.footprint.network/)
User case: 

Logo: 
<img src="https://user-images.githubusercontent.com/19163424/174307906-875f38e1-34d5-4797-a808-34ab4f625ae8.png" width="40%" height="auto">

**Comment by varunprakashnkl** [comment]
Company: Unistam
Website: https://unistamts.com
User Case: Apache Doris is commonly used at Unistam for real-time analytics and high-performance data warehousing. Its scalability and ease of use make it an effective solution for handling large-scale analytical workloads with low-latency query performance.

Logo:
![unistam](https://github.com/user-attachments/assets/0061c321-1646-4e1c-b555-e353b4a077e0)
